packages/sdk/src/single.ts

```diff
@@ -310,35 +318,40 @@ export class RegionRatelimit extends Ratelimit<RegionContext> {
         local bucket = redis.call("HMGET", key, "updatedAt", "tokens")
         
         if bucket[1] == false then
-          -- The bucket does not exist yet, so we create it and add a ttl.
+          -- The bucket does not exist yet, so we create it.
           remaining = maxTokens - 1
           
           redis.call("HMSET", key, "updatedAt", now, "tokens", remaining)
-          redis.call("PEXPIRE", key, interval)
```

Comment:
This does not take care of the expiration after the bucket creation, since we return early here. That means, if the user creates buckets but never takes a token out of them again, we will be leaking buckets

Reply:
I was trying to operate on the existing version and this slipped by, sorry.